frontend/projects/upgrade/src/app/core/feature-flags/feature-flags.service.ts

```diff
@@ -20,6 +20,27 @@ export class FeatureFlagsService {
   isAllFlagsFetched$ = this.store$.pipe(select(selectIsAllFlagsFetched));
   activeDetailsTabIndex$ = this.store$.pipe(select(selectActiveDetailsTabIndex));
 
+  formatDateString(dateString, dateTime) {
```

Comment:
we have `dayjs` for date formatting:
`const date = dayjs("2024-05-31T21:54:19.692Z");
date.format('MMM DD, YYYY, hh:mm A');`

But we also already have a `FormatDate` pipe. We should probably just use that to be consistent with how this and other dates are done. I wouldn't worry too much about getting it exactly like in the design, but if you want to create an additional format, we should use the pipe to do that as well.

```
<span class="ft-14-700 flag-create-date">
        {{ ('home.view-experiment.created-on.text' | translate) + (flag.createdAt | formatDate: 'medium date') }}
      </span>
```

Reply:
addressed this cmt

Reply:
cool, this function can be deleted then.

Reply:
done